storage/storage-interface/src/cached_state_view.rs

```diff
@@ -104,13 +111,20 @@ impl CachedStateView {
     }
 
     pub fn prime_cache_by_write_set(&self, write_sets: &[WriteSet]) -> Result<()> {
-        write_sets
-            .iter()
-            .flat_map(|write_set| write_set.iter())
-            .map(|(key, _)| key)
-            .collect::<HashSet<_>>()
-            .into_iter()
-            .try_for_each(|key| self.get_state_value(key).map(|_| ()))
+        IO_POOL.scope(|s| {
```

Comment:
1. why this not par_iter()?
2. this uses the SyncProofFetcher, does it matter? (should not)

Reply:
1. It's IO bound.. so the bottleneck will be the get_state_value itself.
2. no